apps/web-next/src/app/%5Fhealth/route.tsx

```diff
@@ -0,0 +1,5 @@
+import { type NextRequest } from "next/server";
```

Comment:
Is %5F intentional in path?

Reply:
![image](https://github.com/dcramer/peated/assets/43549299/c4664f0a-2a14-4d1c-952d-f100e3d8f26c)
Maybe it's smth with encoding in mobile gh app

Reply:
Intentional - only way to escape an _ in path according to docs. Eg the path is ‘_health’